en/core-libraries/form.rst

```diff
@@ -130,6 +130,9 @@ Values set with this method will overwrite existing data in the form object::
                     'name' => 'John Doe',
                     'email' => 'john.doe@example.com'
                 ]);
+                // Get the values like this
+                $allValues = $contact->getData();
```

Comment:
I don't know if this example makes sense to put here. It's not related to the rest of the code.

Reply:
It could be in a separate example?

Reply:
I would make them separate if it's important to include.

Reply:
ok, sure. let me create a separate getData sample.